RecoTauTag/HLTProducers/src/L2TauTagFilter.cc

```diff
@@ -35,6 +36,7 @@ class L2TauTagFilter : public HLTFilter {
         ->setComment("Which trigger should the L1 Taus collection pass");
     desc.add<edm::InputTag>("L2Outcomes", edm::InputTag(""))->setComment("L2 CNN outcomes");
     desc.add<double>("DiscrWP", 0.1227)->setComment("value of discriminator threshold");
+    desc.add<double>("l1TauPtThreshold", 250)->setComment("value of discriminator threshold");
```

Comment:
```suggestion
    desc.add<double>("l1TauPtThreshold", 250)->setComment("pT value above which L1T Taus pass the filter regardless of their discriminator output");
```

Reply:
For this copy/paste mistake, the same reasoning as [this comment](https://github.com/cms-sw/cmssw/pull/36009/files#r744107981) applies. It's not a strict requirement for this PR.

Reply:
I will fix it, too.